implementation.js

```diff
@@ -157,6 +157,8 @@ module.exports = function from(arrayLike) {
 	if (arrayFromIterable) {
 		items = arrayFromIterable;
 		len = arrayFromIterable.length;
+	} else if (typeof arrayLike === 'string') {
+		return items.match(/[\uD800-\uDBFF][\uDC00-\uDFFF]?|[^\uD800-\uDFFF]|./g) || [];
```

Comment:
When strings are iterable, they must be parsed by `arrayFromIterable` function. So string splitting using the iterator must be ensured in that function. In this code, my added code will be skipped when strings are iterable.